### DISCUSSION TITLE
Remove default separators when downloading multiple files

### DISCUSSION THREAD

**Discussion by jonny112** [curl/curl#8257] [General]
When downloading multiple files to stdout, like:
`curl https://example.com/split-file.part[1-3] > original-file`
By default, separators of the form:
`--_curl_--https://example.com/next-url`
are inserted in the output, before each transfer.

Naturally, I would have expected the output of the above command to be identical to:
```
curl https://example.com/split-file.part1 > original-file
curl https://example.com/split-file.part2 >> original-file
curl https://example.com/split-file.part3 >> original-file
```

The behavior is currently not documented anywhere and I only found the cause by binary comparison of the output and then searching through the source code. The only (also undocumented) way to disable these separators is using the `--silent` option, but this inevitably also mutes all log output.

I recently suggested an additional option for disabling the separators specifically (https://github.com/curl/curl/pull/8046).
Introducing a new option for this was however seen unjustified.

As the discussion on the PR remained inconclusive, I'd like to suggest to completely remove the separators.
They have more the feel of a debug feature to me than something that could be of productive use.
Instead the unsolicited insertion of data in the output stream is prone to lead to corrupted results.
If required, custom separators can be generated using the `--write-out` option.

If the default separators should be kept for backwards compatibility or other reasons, they should at least be specifically noted in the documentation section about handling multiple URLs.

**Comment by bagder** [comment]
The responses I saw there were all mostly positive to remove the separators, without any additional option. I think we can do that for the reasons you state in here.
